src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/ProfessionalExternalUserFunctionalTest.java

```diff
@@ -73,30 +81,47 @@ public void testExternalUserScenario() {
     }
 
     public void setUpOrgTestData() {
-        superUserEmail = generateRandomEmail();
-        organisationCreationRequest = createOrganisationRequest()
-                .superUser(aUserCreationRequest()
-                        .firstName(firstName)
-                        .lastName(lastName)
-                        .email(superUserEmail)
-                        .build())
-                .build();
-
-        organisationCreationRequest.setStatus("ACTIVE");
-        extActiveOrgId = createAndctivateOrganisationWithGivenRequest(organisationCreationRequest, hmctsAdmin);
-
-        Map<String, Object> searchResponse = professionalApiClient
-                .searchOrganisationUsersByStatusInternal(extActiveOrgId, hmctsAdmin, OK);
-        List<Map<String, Object>> professionalUsersResponses = (List<Map<String, Object>>) searchResponse.get("users");
-        superUserId = (String) (professionalUsersResponses.get(0)).get("userIdentifier");
-    }
-
-    public void setUpUserBearerTokens() {
-        pumBearerToken = inviteUser(puiUserManager);
-        pcmBearerToken = inviteUser(puiCaseManager);
-        pomBearerToken = inviteUser(puiOrgManager);
-        pfmBearerToken = inviteUser(puiFinanceManager);
-        caseworkerBearerToken = inviteUser(caseworker);
+        if (isEmpty(extActiveOrgId)) {
```

Comment:
any reason why we are making this check here?

Reply:
this is to ensure if mistakenly setUpOrgTestData is called then it will not again set up data .This to minimise test data in ref data and sidam

Reply:
Thanks for the explanation @shreedhar-hmcts